### PR TITLE
Issue 341 - Refactoring CssGen.hs

### PR DESCRIPTION
**Pull request by pkukulak**
#341. Thankfully, CssGen was written in a nice, modular fashion and so this was not nearly as difficult as I anticipated it to be. 

I have moved constants into a file name Consts.hs (might be a questionable name now that I think of it), and all functions that generate CSS for different pages into respective files: namely, the new files are AboutCssGen.hs, TimetableCssGen.hs, AboutCssGen.hs and GraphCssGen.hs.

I don't doubt that there are some rough edges to polish in these files.

### CODE REVIEW

File: hs/AboutCssGen.hs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module AboutCssGen where
+
+import Clay
+import Prelude hiding ((**))
+import Data.Monoid
+import Data.Text.Lazy
+import System.Directory
```
Comment (david-yz-liu) [inline hs/AboutCssGen.hs]: Please double-check the imports here... I'm not sure all of them are necessary.

File: hs/CssGen.hs [hunk 2]
```diff
@@ -7,14 +7,18 @@ import Prelude hiding ((**))
 import Data.Monoid
 import Data.Text.Lazy
 import System.Directory
-
+import Consts
+import CommonCssGen as COMMON
```
Comment (david-yz-liu) [inline hs/CssGen.hs]: Not big on the all-caps here. Just import the one style per module, e.g.

`import CommonCssGen (common)`

**Comment by david-yz-liu** [comment]
Looks good, but let's drop the "Gen" from file names.

**Comment by pkukulak** [comment]
I agree on the caps. Fixed.

Files have also been renamed.

Note that since the search symbol pull request has been merged, I made the according changes in TimetableCss.hs as well.

**Pull request merged**